src/DynamoCore/Search/Legacy/BrowserInternalElementForClasses.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System.Collections.ObjectModel;
```

Comment:
I remember this file has been removed from the `master`, if that's right, could you please remove this and its entry in the `*.csproj` file (if there is one).